### DISCUSSION TITLE
Orkes conductor community future

### DISCUSSION THREAD

**Discussion by RussellTaylor83** [orkes-io/orkes-conductor-community#54] [General]
Hi all, 

What is the plan for orkes conductor community? I see Conductor OSS is being updated (6 releases in 6 months), and some of the Orkes devs are working on it. 

Is there a plan to bring orkes conductor community up to date? Or should we be moving to conductor oss?

Thanks

**Comment by ryanmurf** [comment]
@RussellTaylor83 have you learned anything?

**Comment by RussellTaylor83** [comment]
No, nothing at all I'm afraid. We're currently running Orkes from thr most recent February commit.
